### PR TITLE
Base page updates

### PR DESCRIPTION
**Pull request by jleong01**
added line to include banner for school page and coursepage

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/page/basepage/content.html [hunk 1]
```diff
@@ -1 +1,4 @@
-<div data-sly-resource="${'par' @resourceType='wcm/foundation/components/responsivegrid'}"></div>
+<!-- Added validation to change parsys path to what is set on referencePage property -->
+<sly data-sly-test.referenceParsys="${properties.referencePage ? [properties.referencePage,'/jcr:content/par'] : 'par' @ join='/' }"/>
```
Comment (bsinghal1) [inline ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/page/basepage/content.html]: put this into the head.html

File: ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/page/basepage/body.html [hunk 2]
```diff
@@ -23,9 +23,9 @@
 	<sly data-sly-test.mode="${templateName == '/apps/taylorsuniversity/templates/homepage' ? 'edit' : 'disabled'}">
 
           <section class="container-fluid">
-
-	        <sly data-sly-resource="${ @path='header', resourceType='taylorsuniversity/components/content/header', wcmmode=mode}"></sly>
 	        <div class="row">
+		        <sly data-sly-resource="${ @path='header', resourceType='taylorsuniversity/components/content/header', wcmmode=mode}"></sly>
+
```
Comment (bsinghal1) [inline ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/page/basepage/body.html]: Have u verified this change? Please create new pages through homepage & contentpage templates after this change.

Comment (jleong01) [reply ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/page/basepage/body.html]: an email was sent to you regarding this update. subject RE: Regarding the UI issue ...

File: ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/page/basepage/content.html (hunk 1 above)
Comment (jleong01) [reply ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/page/basepage/content.html]: code snippet has been moved to head.html

Comment (jleong01) [reply ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/page/basepage/content.html]: revert back . as code snippet not working if put to head.html

File: ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/page/basepage/content.html [hunk 3]
```diff
@@ -1 +1,3 @@
-<div data-sly-resource="${'par' @resourceType='wcm/foundation/components/responsivegrid'}"></div>
+<sly data-sly-test.referenceParsys="${properties.referencePage ? [properties.referencePage,'/jcr:content/par'] : 'par' @ join='/' }"/>
```
Comment (bsinghal1) [inline ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/page/basepage/content.html]: Can you please elaborate the objective of this change? I mean what will be the difference bring by this change?

**Pull request merged**